sig-advocacy-and-outreach/summer-ospp/README.md

```diff
@@ -0,0 +1,25 @@
+KubeSphere in [Summer 2021 of Open Source Promotion Plan](https://github.com/summer-ospp). You can find our proposals in this page. Or you also can submit your proposal to KubeSphere community.
+
+## Proposals
+
+| Project | Area | Skills to study/improve |
+| --- | --- | --- |
+| TODO | TODO | TODO |
```

Comment:
We can reuse the proposals from GSoC. Because we have not been accepted.

Reply:
Has been reused